automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSpec.scala

```diff
@@ -60,13 +60,14 @@ class LeonardoSpec extends FreeSpec with Matchers with Eventually with ParallelT
                  clusterName: ClusterName,
                  googleProject: GoogleProject,
                  gcsBucketName: GcsBucketName,
+                 creator: WorkbenchEmail,
```

Comment:
So we're refer to person who creates the cluster as "creator" here and "owner" in Sam - is it alright to have two words? (I don't really care if we have 2 different words for it but thought I'd bring it up)

Reply:
solved with faces: in sam it will be called creator